src/pages/Notifications/List/Page.tsx

```diff
@@ -24,10 +25,21 @@ const isBundleStatus = (bundle: Facet | BundleStatus): bundle is BundleStatus =>
 export const NotificationsListPage: React.FunctionComponent = () => {
 
     const params = useParams<NotificationListPageParams>();
+    const history = useHistory();
+    const insights = getInsights();
+    const onFunction = insights?.chrome?.on;
 
     const getBundles = useGetBundles();
     const getApplications = useGetApplications();
 
+    React.useEffect(() => {
+        if (onFunction) {
+            onFunction('APP_NAVIGATION', (event: any) => {
```

Comment:
@Hyperkid123 @karelhala Is there a way to unsubscribe to this event? Does it affect other apps with chrome v2?